ui/app/src/modules/Preview/Tools/ToolPanel.tsx

```diff
@@ -71,10 +72,11 @@ export function ToolPanel(props: ToolPanelProps): ReactElement | null {
   const {
     title,
     BackIcon,
-    onBack = () => dispatch(selectTool())
+    onBack = () => dispatch(selectTool()),
+    classes
   } = props;
   return (
-    <>
+    <div className={classes}>
```

Comment:
Do you really need to wrap the whole ToolPanel? Wouldn't it suffice with applying the class to the body?